The Shipyard.txt

```diff
@@ -0,0 +1,58 @@
+
+
+
+
+
+package gameComponents;
+
+import java.lang.reflect.Array;
+import java.util.ArrayList;
+
+public class Ships {
+public class Faction {
+
+    private final String size;
+    private final String name;
+    private final String faction;
+    private final int Hull;
```

Comment:
Don't capitalize variables.  Its not a requiremnt of the language, but it is convention because it makes it a lot easier to read code later on.  Objects are given capital names, ie Ship, HullZone, Squadron, String.  If you have a capitalized variable, it looks like an object type.  If I violated that guideline in the corresponding SquadronFactory class, please feel free to call it out.

Reply:
This is particularly important on your `Upgrade` fields here.  We are likely to have an `Upgrade` object at some point that is interacting with the `Ship` objects.

It is common convention to include the type of the variable or to err on the side of overly descriptive.  For instance squadValue, so that we don't get confused with the Squadron object, or upgradeList so that we know what kind of object it is.

Reply:
Also I am going to use the squadron builder version of defensetokens, so it is the same for both.